### PR TITLE
Feature/204/show max number in metricchooser

### LINKED ISSUE #204: Feature/Show max number in MetricChooser select-list
# Feature/Show max number in MetricChooser select-list

## Description

Show the max of each metric in braces behind the metric's name in the metric selection drop down. this way you could easily identify interesting and noticeable metrics in a (still) unkown code base. You can also identify metrics that are not interesting, because the maximum is low.

### PR DESCRIPTION
**Pull request by alschmut**
# Feature/204/show max number in metricchooser

closes #204 
## Description

- Show maxValue of each metric in metricChooser select list
- The maxValue doesn't show up next to the selected metrics to avoid any confusions with the current hovered buildings metric
- Add `metricData` attribute to `dataService.data` object to hold the maxValue of each metric. It also allows further data to be stored here (e.x. if the metric is even used in the current shown `renderMap` or if its just in one of the other imported `revisions`)

### Ideas
- I actually wanted to color the maxValue gray or make it smaller, but I didn't find a way to overwrite the CSS settings of the `md-select md-option` AngularMaterial tag...

image

## Definition of Done

A task/pull request will not be considered to be complete until all these items can be checked off.

* [x] **All** requirements mentioned in the issue are implemented
* [x] Does match the Code of Conduct and the Contribution file 
* [x] Task has its own **GitHub issue** (something it is solving)
  * [x] Issue number is **included in the commit messages** for traceability
* [x] **Update the README.md** with any changes/additions made
* [x] **Update the CHANGELOG.md** with any changes/additions made
* [x] **Enough test coverage to ensure that uncovered changes do not break functionality**
* [x] **All tests pass**    
* [x] **Descriptive pull request text**, answering:
  + What problem/issue are you fixing?
  + What does this PR implement and how? 
* [ ] **Assign your PR to someone** for a code review
  + This person _will be contacted **first**_ if a bug is introduced into `master`
* [x] **Manual testing** did not fail

### CODE REVIEW

File: visualization/app/codeCharta/ui/metricChooser/metricChooser.component.scss [hunk 1]
```diff
@@ -38,4 +38,13 @@ color-metric-chooser-component {
     float: right;
   }
 
+  .md-select-value .metric-max-value {
+    display: none;
+  }
+
+}
+
+.metric-max-value {
```
Comment (mrpatpat) [inline visualization/app/codeCharta/ui/metricChooser/metricChooser.component.scss]: you can change the color and size according to your concept here. Can be merged in my opinion. Looks good !

Comment (alschmut) [reply visualization/app/codeCharta/ui/metricChooser/metricChooser.component.scss]: @mrpatpat Yea I tried that, but when running my local server it doesn't adapt the style. Not in chrome, Firefox nor Safari. Did it work for you?

Comment (mrpatpat) [reply visualization/app/codeCharta/ui/metricChooser/metricChooser.component.scss]: Yes, it did. That is strange.

Comment (alschmut) [reply visualization/app/codeCharta/ui/metricChooser/metricChooser.component.scss]: As its not really a problem, I would suggest to merge it. Maybe we will find the solution later.

Comment (alschmut) [reply visualization/app/codeCharta/ui/metricChooser/metricChooser.component.scss]: btw: I just merged the master into this branch and now the grey color kinda shows up on my local server. Maybe... I forgot to pull your changes 😄

**Pull request merged**